src/shared/actions/members.js

```diff
@@ -5,6 +5,7 @@
 import { createActions } from 'redux-actions';
```

Comment:
`actions/members` moved to `topcoder-react-lib`: https://github.com/topcoder-platform/topcoder-react-lib/blob/master/src/actions/members.js
these code updates should be moved there.